packages/graph/hash_graph/lib/graph/src/ontology/data_type.rs

```diff
@@ -35,6 +35,28 @@ impl QueryRecord for DataType {
     type Path<'q> = DataTypeQueryPath;
 }
 
+impl OntologyPath for DataTypeQueryPath {
+    fn base_uri() -> Self {
```

Comment:
Should we suffix with `_path` for these methods? If we were to put these on another trait at some point, it would make the names a little more versatile

Reply:
I'm unsure about that as this is only implemented for paths. It would then be called like `DataTypeQueryPath::base_uri_path()`, or in this use case: 
```rust
FilterExpression::Path(<T::Path<'q>>::version_path())
```

Reply:
Okay, that's a good point, let's not do that, then